query/build.gradle

```diff
@@ -0,0 +1,40 @@
+// Build instructions for standalone Java library.
+
+apply plugin: 'java'
+
+// Set the source and target to Java 6.
+tasks.withType(JavaCompile) {
+    options.compilerArgs << "-source" << "1.6" << "-target" << "1.6"
+}
+
+dependencies {
+    compile project(':core')
+    compile project(':java')
+//    testCompile project(':core')
+    testCompile 'com.fasterxml.jackson.core:jackson-databind:2.3.0'
+    testCompile 'commons-io:commons-io:2.4'
+    testCompile 'junit:junit:4.11'
+    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.mockito:mockito-core:1.9.5'
+//    testCompile files(project(':core').sourceSets.test.output.classesDir)
+    testCompile project(':core').sourceSets.test.output
+
+}
+
+jar {
+    manifest {
+        attributes("Implementation-Vendor": "Keen.io", "Implementation-Version": version)
+    }
+
+    from {
+        project(':core').sourceSets.main.output
+    }
+}
+
+test {
+    include '**/*Test.class'
+}
+
+// archivesBaseName = "keen-client-api-query”
```

Comment:
Question: Why is this commented out? Should it be? I would have thought that we probably want it in there. This is also a question that gets into how we'll be shipping this, though, which we can probably leave as out of scope for the purposes of this review.